calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -73,9 +73,13 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
 
 ### Configure a cluster for WAF
 
-#### Step 1: Configure ApplicationLayer CRD
+#### Step 1: Enable WAF in your cluster
```

Comment:
Thinking about it, we don't want to update the CC docs in this release, because these UI changes will not be included in CC until it starts using 3.16 in about 3-4 months time?

Reply:
Based on where they're located right now, they'll land in CC/3.16.

Reply:
I have reverted docs changes for CC.